src/biokbase/narrative/jobs/job.py

```diff
@@ -451,13 +449,10 @@ def output_state(self, state=None) -> dict:
                 # update timestamp if there was an error
                 state.update({"updated": int(time.time())})
 
-        job_state = {
-            "state": state,
-            "widget_info": widget_info,
-            "user": self.user,
-            "cell_id": self.cell_id,
+        return {
+            "jobState": state,
+            "outputWidgetInfo": widget_info,
```

Comment:
The backend `job_status` request returns data in the format that the frontend uses. 🎉